### PR TITLE
[ENG-18] Use RequestContext to log4js appenders

### PR DESCRIPTION
**Pull request by klesgidis**
## Summary
Uses request context and appends `requestContext.logKeys` to the log message.

It is working for both `console` and `json` logs.

PS 1 Another PR will follow for making `requestContext`'s usage, dependent on the NodeJS version you use

PS 2 Logs from careers with `x-request-id: kyriakos`
Screenshot 2020-11-24 at 1 56 56 PM

### CODE REVIEW

File: src/initializers/diamorphosis.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
   };
   config.printLogo = defaultTo(config.printLogo, true);
   config.log = {
-    pattern: '%[[%d] [%p] %c%] %m',
+    pattern: '%[[%d] [%p] %c%] %x{logTracer} %m',
```
Comment (nikostoulas) [inline src/initializers/diamorphosis.ts:22]: 👏

**Review by nikostoulas — APPROVED** [review]
Nice @klesgidis !

File: .vscode/launch.json [hunk 2]
```diff
@@ -82,8 +82,8 @@
       "name": "Mocha Tests",
       "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
       "args": [
-        "--opts",
```
Comment (nikosd23) [inline .vscode/launch.json:85]: Are you sure you want this change?

**Review by nikosd23 — APPROVED** [review]

File: .vscode/launch.json (hunk 2 above)
Comment (klesgidis) [reply .vscode/launch.json:85]: Yes. The debbugger wasn't working with this. It was deprecated

**Pull request merged**